portal/src/router.ts

```diff
@@ -437,7 +446,11 @@ export default function routerFactory(store) {
         if (from.name === null && (to.name === null || to.name == "login")) {
             console.log("nav", "authenticated", store.getters.isAuthenticated);
             if (store.getters.isAuthenticated) {
-                next("/dashboard");
+                if (store.getters.isTncValid != undefined && !store.getters.isTncValid && to.name != "login") {
```

Comment:
This seems awkward to me. Why the test for undefined here? Afaik getters should always be defined.

Reply:
If there isn't an authenticated user, then isTncValid is undefined. 
Maybe null would be better.
This could be changed to false, but then there should be a check if there is an authenticated user.
What do you suggest?